### DISCUSSION TITLE
Maximum file size Media Library Pro

### DISCUSSION THREAD

**Discussion by Sindoweb** [spatie/laravel-medialibrary#3317] [General] [answered]
Hey everyone,

Running into a weird problem.

In my livewire component I have an upload section defined as this:

`<x-media-library-attachment name="attachment" rules="max:50000" />`

But when I try to add a file (PDF) of 20MB it gives me an error that the file needs to be smaller than 48.83MB.
Any idea what I'm overlooking?

<img width="330" alt="image" src="https://github.com/spatie/laravel-medialibrary/assets/1379550/c02e3dc9-009e-413e-90ca-392a08d3ca44">

**Comment by francoism90** [comment]
https://github.com/spatie/laravel-medialibrary/blob/6cebab870e5193e68b802415a18ed6488d79a512/config/media-library.php#L15

Did you increase this value + your server configuration limits?

**Reply by Sindoweb** [reply]
Thank you for your answer Francois.
Yes I did.

max_file_size is at 50MB.
Server is at 256MB.

`'max_file_size' => 1024 * 1024 * 50, // 50MB`

**Reply by francoism90** [reply]
Hmm, are you use the PDF is 20MB in size? Weird.

**Comment by Sindoweb** [comment, marked as answer]
I forgot to change the Livewire settings to allow bigger uploads...
So changed that to 50MB as well and all went smoothly.
